pom.xml

```diff
@@ -32,6 +32,13 @@
             <version>${junit-jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.9.2</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
You mustn't change the pom file (I also did that, and mentor said, that I must delete it locally).